### PR TITLE
`1.13.9` release

### PR DESCRIPTION
**Pull request by twokul**
+ bumps the version to `1.13.9`
+ changes Ember.js version to `1.13.10`
+ changes Ember Data version to `1.13.14`

- [x] remove ic-ajax switch (that will be part of 2.0)
- [x] broccoli-funnel windows fix – https://github.com/broccolijs/broccoli-funnel/issues/53
   - [x] https://github.com/stefanpenner/matcher-collection/pull/2
   - [x] https://github.com/joliss/node-walk-sync/pull/17
fixes https://github.com/ember-cli/ember-cli/issues/5028

/cc @stefanpenner

### CODE REVIEW

**Comment by stefanpenner** [comment]
awesome, pending the broccoli-funnel windows fix, and we should be good to ship.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+### 1.13.9
+
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + changes to `tests/index.html` file. All tests are now in a separate file, [diff](https://github.com/twokul/ember-cli-release-notes/commit/bd5ac542c0d6dd8e095553d6528ec40ae4be6b4e).
+  + default Ember.js version is now at `1.13.10`
+  + default Ember Data version is now at `1.13.14`
+  + `ember-cli-ic-ajax` has been dropped in favour of `ember-ajax`
```
Comment (stefanpenner) [inline CHANGELOG.md]: this should be part of 2.0

**Comment by stefanpenner** [comment]
1 last thing, ember-ajax switch will be part of 2.0

File: blueprints/app/files/bower.json [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
   "name": "<%= name %>",
   "dependencies": {
-    "ember": "2.1.0",
+    "ember": "1.13.10",
     "ember-cli-shims": "0.0.6",
     "ember-cli-test-loader": "0.2.1",
-    "ember-data": "2.1.0",
+    "ember-data": "1.13.14",
     "ember-load-initializers": "0.1.7",
     "ember-qunit": "0.4.13",
```
Comment (stavarotti) [inline blueprints/app/files/bower.json]: Can you bump this to 0.4.15?  We are are currently using the the `wait` helper, which was added in 0.4.14.

**Comment by jcope2013** [comment]
under users mention ember-resolver change needed in app.js and test/helpers/resolver.js

```import Resolver from 'ember/resolver'; ``` to ``` import Resolver from 'ember-resolver'; ```

https://github.com/aptible/dashboard.aptible.com/pull/423

File: blueprints/app/files/bower.json (hunk 2 above)
Comment (stefanpenner) [inline blueprints/app/files/bower.json]: sure, submit PR to master

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,75 @@
...
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
```
Comment (nightire) [inline CHANGELOG.md]: Link is broken, correct one: http://www.ember-cli.com/user-guide/#upgrading

**Comment by twokul** [comment]
@jcope2013 I thought it would be a part of `2.0.0` release

**Comment by jcope2013** [comment]
@twokul it looks like it is currently part of 1.13.9, notice the change of ember-resolver being moved from bower to package.json, it could be changed back to use the bower version if that change is wanted for 2.0.0 only

https://github.com/twokul/ember-cli-release-notes/commit/0cef67c2c16cb0d6feac7945fa622013531013e0

**Comment by stefanpenner** [comment]
> it could be changed back to use the bower version if that change is wanted for 2.0.0 only

yes lets do this cc @twokul

**Comment by twokul** [comment]
@jcope2013 @stefanpenner looks like I was too hasty to release Ember CLI. thanks for the feedback you guys

File: CHANGELOG.md (hunk 3 above)
Comment (twokul) [inline CHANGELOG.md]: @nightire thanks

**Comment by btecu** [comment]
`ember-data 1.13.15` fixes two bugs. Maybe we should use that one.

**Comment by stefanpenner** [comment]
@btecu good call! @twokul can you swap it in?

Also broccoli-merge-trees is now 1.0.0 we should bump that

**Comment by twokul** [comment]
@stefanpenner will do

**Comment by seanpdoyle** [comment]
@twokul was this released outside of this PR?

https://github.com/ember-cli/ember-cli/releases/tag/v1.13.9

**Comment by stefanpenner** [comment]
@seanpdoyle i believe that is part of the release, did we screw something up?

**Comment by seanpdoyle** [comment]
@stefanpenner I would expect this PR to have been merged and pushed before a release tag was cut, but I'm not involved in this process at all so that might be wrong information

**Comment by btecu** [comment]
@seanpdoyle It was merged in `stable-1.13` branch instead of `master`...

**Comment by seanpdoyle** [comment]
:ok_hand: gotcha sounds great

**Comment by stefanpenner** [comment]
@seanpdoyle Ya this PR is just tracking progress on this branch, kindy quirky (but its more or less a backport release)

**Comment by twokul** [comment]
https://github.com/ember-cli/ember-cli/releases/tag/v1.13.9

**Comment by seanpdoyle** [comment]
Out of the box on a fresh `ember init`, I'm seeing:

```
ENOENT: no such file or directory, stat  '/Users/seanpdoyle/src/ember-cli-rails-heroku-example/frontend/tmp/concat_with_maps-input_base_path-Ff2y9AN9.tmp/0/bower_components/ember-resolver/dist/modules/ember-resolver.js'
```

Any ideas?

**Comment by stefanpenner** [comment]
@twokul is fixing

**Pull request closed without merging**